nntrainer/src/conv2d_layer.cpp

```diff
@@ -164,19 +154,20 @@ Tensor Conv2DLayer::backwarding(Tensor derivative, int iteration) {
           derivative.getAddress(b * derivative.getDim().getFeatureLen() +
                                 i * derivative.height() * derivative.width()),
           in_dim, output.data(), stride, 0.0);
-        memcpy(delK.getAddress(b * delK.getDim().getFeatureLen() + j * o_size),
-               output.data(), o_size * sizeof(float));
+        float *del = delK.getAddress(j * o_size);
+        for (unsigned k = 0; k < o_size; ++k) {
+          del[i] += output[i];
+        }
       }
 
       // Calculate delBias [ batch , 1, 1, filter_size]
 
-      float sum = 0.0;
       for (unsigned int j = 0; j < derivative.height(); ++j) {
         for (unsigned int k = 0; k < derivative.width(); ++k) {
           sum += derivative.getValue(b, i, j, k);
         }
       }
-      delBias.setValue(b, 0, 0, 0, sum);
+      delBias.setValue(0, 0, 0, 0, sum);
```

Comment:
Should this be doing addition than just setValue??

Reply:
fixed. thanks.